BeVolt/BSP/Simulator/DataGeneration/Data/CAN.csv

```diff
@@ -0,0 +1 @@
+0x10a,0182153675000000,5
```

Comment:
I'm not exactly sure how this file got into your commit? The gitignore has *.csv on it